diagnostic.md

```diff
@@ -140,6 +156,13 @@ div {
 ```
 
 <!-- your answer starts here -->
+divone: on top right
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).